index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
                             <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
                             <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
                         </ul>
-        </section>  
+        </div>
+    </section>  
+            <!-- the first card............... -->
+            <section id="work-section">
+                <div class="cards">
+                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <div class="align-text">
+                    <h2 class="h2-size">Tonic</h2>
+                    <ul class="first-list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list2">2015</li>
+                    </ul>
+                    <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                    <ul class="blue-list">
+                        <li>HTML</li>
```

Comment:
[OPTIONAL]
- [ ] for the various cards you need to give it some padding as shown in the figma template. notice there are spaces surrounding the contents of each cards, please make this change to your web application :+1: 
- figma design
![f-t-p](https://user-images.githubusercontent.com/89055502/235337897-77825039-2df2-48ec-b09a-bfb19de7a2fd.png)
- your design
![s-t-p](https://user-images.githubusercontent.com/89055502/235337906-80ab697a-ad46-4a6c-9d20-7fdfd9ebea25.png)